src/main/resources/assets/spookytime/lang/de_de.json

```diff
@@ -1,32 +1,28 @@
 {
-    "spookytime.about.github": "Unserer Githublink liegt da: ",
-    "spookytime.github": "https://github.com/fabric-community/spooky-time",
-    
-    "spookytime.cmd.spooktober": "Es ist Spooktober!",
-    "spookytime.cmd.tillspooktober": "Es ist %d Tage bis Spooktober",
-    
-    "item.spookytime.pumpcown_spawn_egg": "Pumpcown-Spawn-Ei",
-    "item.spookytime.blaze_skirt": "Lohenrock",
-    "item.spookytime.spookium_ingot": "Spookiumbarren",
-    "item.spookytime.spookium_nugget": "Spookiumklumpen",
-    "item.spookytime.soul_bottle": "Seeleflasche",
-    "item.spookytime.pumpkin_candy": "K�rbisbonbon",
-    "item.spookytime.rare_candy": "Seltenes Bonbon",
-    "item.spookytime.winged_strawberry_candy": "Gefl�geltes Erdbeerebonbon",
-    "item.spookytime.caramel_apple": "Karamelapfel",
-    "item.spookytime.baked_pumpkin_seeds": "Gebackene K�rbiskerne",
-    
-    "block.spookytime.deceased_grass_block": "Verstorbener Grasblock",
-    "block.spookytime.deceased_dirt": "Verstorbene Erde",
-    "block.spookytime.tainted_gravel": "Verdorbener Kies",
-    "block.spookytime.tainted_sand": "Verdorbener Sand",
-    "block.spookytime.tainted_glass": "Verdorbenes Glas",
-    "block.spookytime.tainted_glass_pane": "Verdorbenes Glasscheibe",
-    "block.spookytime.soul_glass": "Seeleglas",
-    "block.spookytime.soul_glass_pane": "Seeleglasscheibe",
-    "block.spookytime.spookium_ore": "Spookiumerz",
-    "block.spookytime.spookium_block": "Spookiumblock",
-    
-    "biome.spookytime.spooky_forest": "Gespenstischer Wald",
-    "biome.spookytime.spooky_lowlands": "Gespenstisches Flachland"
-}
\ No newline at end of file
+	"spookytime.about.github": "Unserer Githublink liegt da: ",
+	"spookytime.github": "https://github.com/fabric-community/spooky-time",
+	"spookytime.cmd.spooktober": "Es ist Spooktober!",
+	"spookytime.cmd.tillspooktober": "Es ist %d Tage bis Spooktober",
+	"item.spookytime.pumpcown_spawn_egg": "Pumpcown-Spawn-Ei",
+	"item.spookytime.blaze_skirt": "Lohenrock",
+	"item.spookytime.spookium_ingot": "Spookiumbarren",
+	"item.spookytime.spookium_nugget": "Spookiumklumpen",
+	"item.spookytime.soul_bottle": "Seeleflasche",
+	"item.spookytime.pumpkin_candy": "K�rbisbonbon",
```

Comment:
Did the file encoding change? These question marks were `ü` before.

Reply:
Hmm, odd. All I did was run them through Idea's Reformat Code. I'll see if I can fix.